### PR TITLE
Change cypress test from js to ts

### PR DESCRIPTION
**Pull request by allysonjp715**

### CODE REVIEW

**Comment by attwad** [comment]
Can you extend our existing common tsconfig that we share accross all services?
https://github.com/open-covid-data/healthmap-gdo-temp/blob/master/common/tsconfig.json

https://github.com/open-covid-data/healthmap-gdo-temp/blob/master/data-serving/data-service/tsconfig.json#L2

**Comment by allysonjp715** [comment]
> Can you extend our existing common tsconfig that we share accross all services?
> https://github.com/open-covid-data/healthmap-gdo-temp/blob/master/common/tsconfig.json
> 
> https://github.com/open-covid-data/healthmap-gdo-temp/blob/master/data-serving/data-service/tsconfig.json#L2

Done

File: verification/curator-service/ui/cypress/tsconfig.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+    "extends": "../../../../common/tsconfig.json",
     "compilerOptions": {
         "strict": true,
```
Comment (attwad) [inline verification/curator-service/ui/cypress/tsconfig.json:4]: common already has strict: true this is not needed.

File: verification/curator-service/ui/cypress/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+    "extends": "../../../../common/tsconfig.json",
+    "compilerOptions": {
+        "strict": true,
+        "baseUrl": "../node_modules",
+        "target": "es5",
```
Comment (attwad) [inline verification/curator-service/ui/cypress/tsconfig.json:6]: this blog tends to indicate es6 is supported
https://glebbahmutov.com/blog/use-typescript-with-cypress/

can you remove this line and leave it to the common tsconfig to target es6?

you might have to change the "lib" below

File: verification/curator-service/ui/cypress/tsconfig.json (hunk 1 above)
Comment (allysonjp715) [reply verification/curator-service/ui/cypress/tsconfig.json]: Done

File: verification/curator-service/ui/cypress/tsconfig.json (hunk 2 above)
Comment (allysonjp715) [reply verification/curator-service/ui/cypress/tsconfig.json]: Done

**Pull request merged**